dub.sdl

```diff
@@ -29,9 +29,8 @@ subPackage {
     "src/ddmd/tokens.d" \
     "src/ddmd/utf.d"
 
-  // stringImportPaths cannot be used because it will make Dub extremely slow
-  // https://github.com/dlang/dub/issues/1199
-  dflags "-J$PACKAGE_DIR"
+  preGenerateCommands "\"$${DUB_PACKAGE_DIR}config.sh\" \"$${DUB_PACKAGE_DIR}generated/dub\" VERSION /etc"
+  stringImportPaths "generated/dub"
```

Comment:
> No, . does exactly what you want.

I don't know how, but I got this error `Invalid variable: DFLAGS` when using a dot `.`. As you can see in https://github.com/dlang/dub/issues/1199. For some reason I don't get this error anymore today 😕.